flink-test-utils-parent/flink-connector-test-utils/src/main/java/org/apache/flink/connector/testframe/utils/TestDataMatchers.java

```diff
@@ -33,180 +36,194 @@
 
```

Comment:
Actually this `TestDataMatchers` is a little bit hack because it bases on non-public implementation of Hamcrest matchers. 

I made a commit for rewriting this `TestDataMatchers` with AssertJ-style Assertions:
https://github.com/PatrickRen/flink/commit/56d6e0a83d55884f605e8949d0349c0ebf64f2d3

Feel free to cherry-pick it into your PR

Reply:
Thanks @PatrickRen , I will reuse it.